src/components/contact/ContactForm.tsx

```
@@ -0,0 +1,672 @@
+    return (
+      <Alert variant="destructive">
+        <AlertCircle className="h-4 w-4" />
+        <AlertTitle>Configuration Error</AlertTitle>
+        <AlertDescription>
+          hCaptcha site key is missing. Please contact the site administrator.
+        </AlertDescription>
+      </Alert>
+    );
+  }
+
+  const [secret, setSecret] = React.useState<string | null>(null);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)